feature-libs/order/root/model/order.model.ts

```diff
@@ -1,43 +1,11 @@
 import {
-  Address,
```

Comment:
We should remove the whole file.
The goal was to keep things clear and simple like we stated in the ticket description.  
Lets discuss.